kfflib/build.gradle.kts

```diff
@@ -70,6 +70,10 @@ configurations {
     runtimeElements {
         setExtendsFrom(emptySet())
     }
+    api {
```

Comment:
this is required to run `:kfflib:runClient`, without it test mod in kfflib sourceset cannot be launched